packages/core/src/provider/plugins/progress.ts

```diff
@@ -15,11 +15,12 @@ export const pluginProgress = (): RsbuildPlugin => ({
         return;
       }
 
-      const { ProgressPlugin } = await import('@rspack/core');
+      const prefix = options !== true ? options.id : TARGET_ID_MAP[target];
```

Comment:
Thanks for your contribution ❤️

Is `options.id` can be undefined here?

Reply:
undefined will cause no prefix.

![image](https://github.com/web-infra-dev/rsbuild/assets/25167721/74bcc4ef-df83-4672-af0d-7a16d41ce7be)

Reply:
See rspack

![image](https://github.com/web-infra-dev/rsbuild/assets/25167721/d0f05e3d-9376-41c5-b2ed-d4172a8a735f)

Reply:
How about do it as follows, which is more intuitive:

- progressBar.id: undefined -> display the default id
- progressBar.id: '' (empty string) -> display no id

Reply:
Great suggestion:

```ts
const prefix = (options !== true && options.id !== undefined) ? options.id : TARGET_ID_MAP[target];
```